packages/web-app-preview/src/App.vue

```diff
@@ -208,6 +211,8 @@ export default defineComponent({
       }
     ]
 
+    const fileId = unref(currentFileContext).itemId
```

Comment:
When unrefing in the root of the `setup` function, you're losing reactivity. Better make this a `computed`. This way it stays reactive when the fileId changes.

```
const fileId = computed(() => unref(currentFileContext).itemId)
```